docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -11,13 +11,26 @@ locations.
    Azure subscription, as well as the ability to create and configure AAD
    applications.
 
+1. Set the az account
+   ```bash
+   az account set -n "ARO Classic E2E"
+   ```
+
+1. You will need a resource group for global infrastructure
+   ```bash
+   GLOBAL_RESOURCEGROUP=global-infra
+   az group create -n $GLOBAL_RESOURCEGROUP --location eastus
+   ```
+
 1. You will need a publicly resolvable DNS Zone resource in your Azure
    subscription. Set PARENT_DOMAIN_NAME and PARENT_DOMAIN_RESOURCEGROUP to the name and
    resource group of the DNS Zone resource:
 
    ```bash
-   PARENT_DOMAIN_NAME=osadev.cloud
-   PARENT_DOMAIN_RESOURCEGROUP=dns
+   PARENT_DOMAIN_NAME=aro-classic.e2e.v4.dev.azmosa.io
+   PARENT_DOMAIN_RESOURCEGROUP=global-infra
+
+   az network dns zone create --name $PARENT_DOMAIN_NAME -g $PARENT_DOMAIN_RESOURCEGROUP
```

Comment:
Did this domain change for local development? We should probably make the values used here generic, and not point to any specific environments' values, regardless.